package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-drupal",
-  "version": "0.0.11",
+  "version": "0.0.12",
```

Comment:
I think this one warrants a minor version bump due to the path change for `bootstrap.js` -> `index.js`, that is potentially breaking if anyone put it in `package.json` scripts.